### PR TITLE
new

### PR DESCRIPTION
**Pull request by AnasXbouali**

### CODE REVIEW

File: docs/src/ho.md [hunk 1]
```diff
@@ -174,10 +174,13 @@ println(" p2(t1+) - p2(t1-) = ", jmp1)
 println(" p2(t2+) - p2(t2-) = ", jmp2)
 ```
 
+## Indirect Method 
+
+
 ```@example main
-using NLsolve
-using Animations
+using NonlinearSolve
```
Comment (ocots) [inline docs/src/ho.md:181]: You have to add NonlinearSolve.jl in the Project.toml of the directory `docs`.

```julia
julia> ]
activate docs
add NonlinearSolve
```

**Comment by ocots** [comment]
@AnasXbouali Why do you close and reopen the PR so many times? 

If you are working on the PR and you want to block merging you can click on "convert to draft" on top of the page.

**Comment by AnasXbouali** [comment]
@ocots I had a problem during the build, the issue was to add the NonlinearSolve package.

**Comment by ocots** [comment]
When you are ready to merge, you can click on "Ready for review" at the bottom of the page.

**Pull request merged**